draft-mouris-cfrg-mastic.md

```diff
@@ -369,13 +449,11 @@ def gen(self,
         ) -> tuple[list[CorrectionWord], list[bytes]]:
```

Comment:
Text above this point (circa 391) is now out of date: the one-hotness, path, and counter consistency checks have been moved out of VIDPF into Mastic. If you'd like to update the algorithm definitions in this PR, then it would be helpful if you could also align the prose description with the spec.

Reply:
Revised description in Mastic